index.html

```diff
@@ -552,7 +552,42 @@ <h2>Creators</h2>
+            
+            <b>What will you get if you join us?</b> Opportunity to:
+            <ul class="Join_Us_list1">
+              <li>build AI-driven and educational projects like Minerva or Aibyss</li>
+              <li>develop full-stack, DevOps, or ML practical skills through hands-on collaboration</li>
+              <li>learn time-saving workflows (e.g., CI/CD, Agile practices) to boost productivity</li>
+              <li>get meaningful feedback during weekly peer code reviews and bi-weekly community meetings to boost your professional growth</li>
+              <li>lead projects and mentor others</li>
+            </ul>
+            
+            <b>Is there a fee?</b> While the opportunities you receive may be expensive elsewhere, joining our community costs $0 USD. This lets us, unlike other communities and courses, value quality of the community members over quantity. We believe all of us can learn from each other regardless of experience or seniority.<br>
+            
+            <p class = "What_expected">What is expected from you?</p> To maintain our $0 USD membership, we ask you to contribute regularly. <b>Every</b> community member is expected to participate in the community projects (or lead new ones), attend and host community meetings and webinars, and participate in and organize community events. We take this very seriously. Members typically contribute 4–6 hours weekly. The best part is that the actual learning, growth, and development happen when you actively do all of these things!<br>
+            
+            <p class="how_to_join">How to join?</p>
+            <ol class="Join_Us_list2">
+              <li><b>Find an issue:</b> Browse <a href="https://github.com/move-fast-and-break-things">our GitHub projects</a> and look for “good first issue” labels.</li>
+              <li><b>Claim it:</b> Comment on the issue to let us know you’re working on it.</li>  
+              <li><b>Code:</b> Submit a PR with your solution.</li>  
+              <li><b>Collaborate:</b> Work with us to refine and merge your PR.</li>  
+              <li><b>Join:</b> We’ll email you within 48 hours after merging by email associated with your Git commits!</li>
+            </ol>
+            Ready to contribute to the community and believe we will have fun working on cool projects together? Start by picking your first GitHub issue today!
+            
+            <p class="Join_Us_lines">If you have any questions, contact us on <a href="mfbt.community@gmail.com">mfbt.community@gmail.com.</a></p></div> 
```

Comment:
Так как это ссылка на email адрес, она должна начинаться с `mailto` – иначе, не будет работать.

```suggestion
            <p class="Join_Us_lines">If you have any questions, contact us on <a href="mailto:mfbt.community@gmail.com">mfbt.community@gmail.com.</a></p></div> 
```